drivers/sensor/ite_vcmp_it8xxx2/Kconfig

```diff
@@ -0,0 +1,11 @@
+# ITE Voltage Comparator driver configuration options
+
+# Copyright (c) 2022 ITE Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config VCMP_IT8XXX2
```

Comment:
Since this is an on-chip peripheral, I think this should depend on `SOC_SERIES_RISCV32_IT8XXX2`; it's not relevant if not targeting IT8xxx2.

Reply:
Done